iothub/service/src/Exceptions/ExceptionHandlingHelper.cs

```diff
@@ -18,102 +17,30 @@ internal class ExceptionHandlingHelper
-            {
-                HttpStatusCode.Forbidden,
-                async (response) => new QuotaExceededException(
-                    code: await GetExceptionCodeAsync(response).ConfigureAwait(false),
-                    message: await GetExceptionMessageAsync(response).ConfigureAwait(false))
-            },
-            {
-                HttpStatusCode.PreconditionFailed,
-                async (response) => new DeviceMessageLockLostException(
-                    code: await GetExceptionCodeAsync(response).ConfigureAwait(false),
-                    message: await GetExceptionMessageAsync(response).ConfigureAwait(false))
-            },
-            {
-                HttpStatusCode.RequestEntityTooLarge,
-                async (response) => new MessageTooLargeException(
-                    code: await GetExceptionCodeAsync(response).ConfigureAwait(false),
-                    message: await GetExceptionMessageAsync(response).ConfigureAwait(false))
-            },
-            {
-                HttpStatusCode.InternalServerError,
-                async (response) => new ServerErrorException(
-                    code: await GetExceptionCodeAsync(response).ConfigureAwait(false),
-                    message: await GetExceptionMessageAsync(response).ConfigureAwait(false))
-            },
-            {
-                HttpStatusCode.ServiceUnavailable,
-                async (response) => new ServerBusyException(
-                    code: await GetExceptionCodeAsync(response).ConfigureAwait(false),
-                    message: await GetExceptionMessageAsync(response).ConfigureAwait(false))
-            },
-            {
-                (HttpStatusCode)429,
-                async (response) => new ThrottlingException(
-                    code: await GetExceptionCodeAsync(response).ConfigureAwait(false),
-                    message: await GetExceptionMessageAsync(response).ConfigureAwait(false))
-            }
-        };
-
-        public static IReadOnlyDictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>> GetDefaultErrorMapping() =>
-            s_mappings;
-
         public static Task<string> GetExceptionMessageAsync(HttpResponseMessage response)
         {
             return response.Content.ReadAsStringAsync();
         }
 
         /// <summary>
-        /// Get the fully-qualified error code from the HTTP response message, if exists.
+        /// Get the fully-qualified error iotHubStatusCode from the HTTP response message, if exists.
         /// </summary>
         /// <param name="response">The HTTP response message</param>
-        /// <returns>The fully-qualified error code, or the response status code, if no error code was provided.</returns>
-        public static async Task<ErrorCode> GetExceptionCodeAsync(HttpResponseMessage response)
+        /// <returns>The fully-qualified error iotHubStatusCode, or the response status iotHubStatusCode, if no error iotHubStatusCode was provided.</returns>
```

Comment:
Lines in this method are getting quite long. Can you make them shorter where excessively long?